osu.Game.Rulesets.Osu/Difficulty/Skills/OsuSkill.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using osu.Game.Rulesets.Difficulty.Preprocessing;
+using osu.Game.Rulesets.Difficulty.Skills;
+using osu.Game.Rulesets.Mods;
+using System.Linq;
+
+namespace osu.Game.Rulesets.Osu.Difficulty.Skills
+{
+    public abstract class OsuSkill : StrainSkill
+    {
+        public OsuSkill(Mod[] mods) : base(mods)
+        {
+        }
+
+        public double OsuDifficultyValue()
```

Comment:
I think it would be better to allow overriding of `DifficultyValue` in `StrainSkill` and changing this into `public double override DifficultyValue()`

Reply:
There needs to be some way to modify `strains` as well in that case.